src/dmd/cli.d

```diff
@@ -68,18 +68,66 @@ bool isCurrentTargetOS(TargetOS os)
 }
 
 /**
-Contains all available CLI $(LREF Option)s.
+Contains all available CLI $(LREF Usage.Option)s.
 
-See_Also: $(LREF Option)
+See_Also: $(LREF Usage.Option)
 */
 struct Usage
 {
-    /// Representation of a CLI `Option`
+    /**
+    * Representation of a CLI `Option`
+    *
+    * The DDoc description `ddoxText` is only available when compiled with `-version=DdocOptions`.
+    */
     struct Option
     {
         string flag; /// The CLI flag without leading `-`, e.g. `color`
         string helpText; /// A detailed description of the flag
         TargetOS os = TargetOS.all; /// For which `TargetOS` the flags are applicable
+
+    version(DdocOptions)
+    {
+        string ddocText; /// Detailed description of the flag (in Ddoc)
+
+        this(string flag, string helpText, TargetOS os = TargetOS.all)
```

Comment:
Maybe pull this method out of the version blocks since it appears to be the same in both cases?